### PR TITLE
[PT-BR] Quick Fixes.ts, Fixits.ts and Mobile Support.ts

### PR DESCRIPTION
**Pull request by rogerpoliver**
@khaosdoctor @alvarocamillont @danilofuchs @orta

Add PT-BR translations to:

- Quick Fixes.ts
- Fixits.ts
- Mobile Support.ts

### CODE REVIEW

File: packages/playground-examples/copy/pt/JavaScript/Helping with JavaScript/Quick Fixes.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// O Typescript fornece recomendações rápidas para problemas comuns.
+// Os comandos apareceram no seu editor com base nas recomendações.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/JavaScript/Helping with JavaScript/Quick Fixes.ts]:
```suggestion
// Os comandos aparecem no seu editor com base nas recomendações.
```

File: packages/playground-examples/copy/pt/Playground/Language/Fixits.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+// Provavelmente isso é algo que você não vai usar
+// no playground quando estiver aprendendo ou testando exemplos.
+
+// De qualquer maneira, ter fixits disponíveis significa que
+// podemos documentá-las no playground e isso é valioso.
+
+// exemplo:big-number-literals
+// exemplo:const-to-let
+// exemplo:infer-from-usage-changes
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/Playground/Language/Fixits.ts]:
```suggestion
// example:big-number-literals
// example:const-to-let
// example:infer-from-usage-changes
```

This syntax: `example:*` is for hyperlinks, and must not be translated!

**Review by danilofuchs — CHANGES_REQUESTED** [review]
Great job! I have some minor suggestions

**Comment by rogerpoliver** [comment]
> Great job! I have some minor suggestions

Thanks! I already commited your suggestions.

**Pull request merged**